yesterdaysGames.py

```diff
@@ -0,0 +1,60 @@
+"""
+Gray Goolsby
+8/5/2018
+
+Pulls game results from previous day
+"""
+import pandas as pd
+from pullWebData import pullSoup
+
+def pullGameTable(url):
+    soup = pullSoup(url)
+    
+    allTables = soup.findAll('div', {"class": "game_summaries"})
+    gameTables = allTables[0].findAll('table')
+    games = []
+
+    for i in range(len(gameTables)):
+        gameStats = gameTables[i].findAll('td')
```

Comment:
```
for element in gameTables: 
    gameStats = element.findAll('td')
```
You could do something like that instead- a little cleaner

Reply:
can't because I use i as tracker to distinguish between games and pitchers